src/reducers/events/event-list-reducer.js

```diff
@@ -59,14 +72,25 @@ const eventListReducer = (state = DEFAULT_STATE, action) => {
         term,
         summitTZ,
         filters,
-        extraColumns
+        extraColumns,
+        selectionPlans: selection_plans
       };
     }
     case RECEIVE_EVENTS: {
-      const { current_page, total, last_page } = payload.response;
+      const { data, current_page, total, last_page } = payload.response;
+
+      const events = data.map((e) => ({
+        ...e,
+        selection_plan: e.selection_plan
+          ? state.selectionPlans.find(
+              (sp) => sp.label === e.selection_plan.name
```

Comment:
i do think that is better to use the ID  in case of the selection plan change its name